tests/unit/mutexExecutor.test.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-floating-promises */
```

Comment:
This actually changes how the test works if I'm not mistaken. We should keep it as is.